docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -40,9 +40,10 @@ a selection of packages.
 There are different distributions available:
 
 * {version} (recommended): +docker.elastic.co/package-registry/distribution:{version}+ - Selection of packages from the production repository released with the {version} version of the {stack}.
+* lite {version}: +docker.elastic.co/package-registry/distribution:lite-{version}+ - Subset of the most commonly used packages from the production repository released with the {version} version of the {stack}.
 * production: `docker.elastic.co/package-registry/distribution:production` - Packages available in the production registry (https://epr.elastic.co).
-* staging: `docker.elastic.co/package-registry/distribution:staging` - Packages available in the staging registry (https://epr-staging.elastic.co). These packages may be pending of validation.
-* snapshot: `docker.elastic.co/package-registry/distribution:snapshot` - Packages under development.
+* lite: `docker.elastic.co/package-registry/distribution:lite` - Small subset of packages available in the production registry (https://epr.elastic.co).
+This image includes only the most commonly used packages, to be used when just common packages are needed or when trying Fleet in air-gapped environments.
```

Comment:
Couple of thoughts: "to be used when just common packages are needed" seems a little vague, and I'm not sure I would say "common" if you mean "commonly used" because there's room for misinterpretation. Also I would change "trying" to something like "running" because "trying" makes it sound like it's difficult to do (which might be true, but let's stay positive in the docs. 😄